packages/wallets/src/components/WalletsPercentageSelector/WalletsPercentageSelector.scss

```diff
@@ -0,0 +1,39 @@
+.wallets-percentage-selector {
+    display: flex;
+    justify-content: space-around;
+    width: 36rem;
+
+    @include mobile {
+        width: 100%;
+    }
+
+    &-block {
+        cursor: pointer;
+        background-color: var(--general-section-1);
```

Comment:
These are not defined yet, pls add a fallback

Reply:
Done